src/TLuaInterpreter.cpp

```diff
@@ -11698,6 +11843,65 @@ bool TLuaInterpreter::call(const QString& function, const QString& mName)
     }
 }
 
+// No documentation available in wiki - internal function
+std::pair<bool, bool> TLuaInterpreter::callReturnBool(const QString& function, const QString& mName)
+{
+    lua_State* L = pGlobalLua;
+    if (!L) {
```

Comment:
You don't check this in the function above, so it's a bit inconsistent.

Reply:
Thanks, fixed.